lib/Controller/MasterController.php

```diff
@@ -79,8 +80,12 @@ public function __construct(
 	public function autoLogout(?string $jwt) {
 		try {
 			if ($jwt !== null && $this->master->isValidJwt($jwt)) {
+				$key = $this->gss->getJwtKey();
+				$decoded = (array)JWT::decode($jwt, new Key($key, Application::JWT_ALGORITHM));
+
+				$idp = $decoded['saml.idp'] ?? 0;
```

Comment:
maybe `null` if it is not set. In case there is a ever 0 as valid number, and also to better distinguish this case for the casual reader.